### PR TITLE
Create Pancakes.is-a.dev.json

### PR DESCRIPTION
**Pull request by pancak3boy**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### Link to Website

https://k.pancakeboy.repl.co

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please read the [docs](https://www.is-a.dev/docs) and update your file with the valid format.

**Comment by pancak3boy** [comment]
image

**Comment by WilliamDavidHarrison** [comment]
> image

Oh uh, I'll just do it for you.

File: domains/pancakes.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "pancak3boy",
+        "email": ""
```
Comment (WilliamDavidHarrison) [inline domains/pancakes.json:4]: Can you put your email here

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/pancakes.json (hunk 1 above)
Comment (pancak3boy) [reply domains/pancakes.json]: done sorry for late reply

**Review by wdhdev — CHANGES_REQUESTED** [review]
Sorry, you need to finish your website before I can merge :(

**Comment by pancak3boy** [comment]
will that do now? (I made a site)

**Review by wdhdev — CHANGES_REQUESTED** [review]
Sorry you need to finish your website

**Comment by pancak3boy** [comment]
I did tho? what else do I need to do now

**Comment by WilliamDavidHarrison** [comment]
> I did tho? what else do I need to do now

The website can't be under development it needs to be finished.

**Comment by pancak3boy** [comment]
I DID THO

**Comment by WilliamDavidHarrison** [comment]
> I DID THO
Screenshot_20230109_065615_Chrome

**Comment by pancak3boy** [comment]
I hate my life.... REPLIT DIDNT PUBLISH MY UPDATE FOR SOME REASON but it should be fixed now

**Review by wdhdev — APPROVED** [review]

**Pull request merged**